src/utils/ConsoleLogger.js

```diff
@@ -8,7 +8,7 @@ class ConsoleLogger {
     }
 
     debug(...args) {
-        if (process.env.DEBUG === 'true') {
+        if (process.env.DEBUG) {
```

Comment:
You shouldn't change that. It does not work now. If process.env.DEBUG = "false" it will still enter that if statement. enviromental variables from .env are always strings. Please don't change this as it is not in a scope of your task.

Reply:
Yeah, sorry with @Veryyapeee we tried to fixed something and I frorgot to change it back, I'm gonna do this